From Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Thu, 7 Jul 2022 22:18:03 +0200
Subject: [PATCH] Replace NULL to PedPartitionFlag static_cast

It fails when NULL is defined as std::nullptr_t.

Error:
GParted_Core.cc: In constructor 'GParted::GParted_Core::GParted_Core()':
GParted_Core.cc:75:64: error: invalid 'static_cast' from type 'std::nullptr_t' to type 'PedPartitionFlag'
   75 |         for ( PedPartitionFlag flag = ped_partition_flag_next( static_cast<PedPartitionFlag>( NULL ) ) ;

---
 src/GParted_Core.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/GParted_Core.cc b/src/GParted_Core.cc
index 46c651933866..6502134d2819 100644
--- a/src/GParted_Core.cc
+++ b/src/GParted_Core.cc
@@ -72,7 +72,7 @@ GParted_Core::GParted_Core()
 	ped_exception_set_handler( ped_exception_handler ) ; 
 
 	//get valid flags ...
-	for ( PedPartitionFlag flag = ped_partition_flag_next( static_cast<PedPartitionFlag>( NULL ) ) ;
+	for ( PedPartitionFlag flag = ped_partition_flag_next( static_cast<PedPartitionFlag>( 0 ) ) ;
 	      flag ;
 	      flag = ped_partition_flag_next( flag ) )
 		flags .push_back( flag ) ;
